pyserini/search/pyquerybuilder.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+#
+# Pyserini: Python interface to the Anserini IR toolkit built on Lucene
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This module provides Pyserini's Python interface query building for Anserini.
+"""
+import logging
+from ..pyclass import JQueryGeneratorUtils, JTermQuery, JTerm, JBoostQuery
+from ..analysis.pyanalysis import get_lucene_analyzer, Analyzer
+
+logger = logging.getLogger(__name__)
+
+
+def get_boolean_query_builder():
+    """ Get a BooleanQueryBuilder object.
+
+    Returns
+    -------
+    JBooleanQueryBuilder
+    """
+    return JQueryGeneratorUtils.getBooleanQueryBuilder()
+
+
+def get_clause_should():
+    """ Get a BooleanClause.Occur.SHOULD statement
+
+    Returns
+    -------
+    BooleanClause.Occur.SHOULD
+    """
+    return JQueryGeneratorUtils.getBooleanClauseShould()
+
+
+def get_clause_must():
```

Comment:
Would it be better to implement an Enum for these? Like here? https://github.com/castorini/pyserini/blob/master/pyserini/pyclass.py#L103

Reply:
Hrm. Wait, or maybe somehow bind them to singleton symbols?

Dunno...